ontology/spdx-ontology.owl.xml

```diff
@@ -9,14 +9,15 @@
      xmlns:doap="http://usefulinc.com/ns/doap#"
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
     <owl:Ontology rdf:about="http://spdx.org/rdf/terms">
-        <owl:versionIRI rdf:resource="http://spdx.org/rdf/terms/2.2"/>
-        <rdfs:comment>This specification describes the SPDX® language, defined as a dictionary of named properties and classes using W3C&apos;s RDF Technology.
-
-SPDX® is a designed to allow the exchange of data about software packages. This information includes general information about the package, licensing information about the package as a whole, a manifest of files contained in the package and licensing information related to the contained files.
-
-Known issues:</rdfs:comment>
-        <rdfs:label>SPDX 2.2</rdfs:label>
-        <owl:versionInfo rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">2.2</owl:versionInfo>
+        <owl:versionIRI rdf:resource="http://spdx.org/rdf/terms/2.3"/>
```

Comment:
I'd use the simpler `http://spdx.org/rdf/2.3`

Reply:
> I'd use the simpler http://spdx.org/rdf/2.3

Previous versions have always used the `rdf/terms` as part of the URI - so to keep it consistent, I would like to leave this as is.